### PR TITLE
feat(api-client): address bar http methods

### PR DESCRIPTION
**Pull request by antlio**
this pr updates http methods style and ordering in the address bar:

**before**
image

**after**
image

### CODE REVIEW

**Review by hanspagel — APPROVED** [review]
Love it! @cameronrohani should take a look, too

File: packages/oas-utils/src/helpers/httpMethods.ts [hunk 1]
```diff
@@ -1,58 +1,60 @@
 import type { RequestMethod } from '@/entities/spec/requests'
 
-export const REQUEST_METHODS: {
-  [x in RequestMethod]: {
-    short: string
-    color: string
+/** HTTP methods in a specific order */
+const ORDERED_METHODS: RequestMethod[] = [
```
Comment (amritk) [inline packages/oas-utils/src/helpers/httpMethods.ts:4]: I would just order these in entitites/spec/requests and put on a comment that they should not be re-ordered. Then we only have one source of truth

File: packages/oas-utils/src/helpers/httpMethods.ts [hunk 2]
```diff
@@ -1,42 +1,21 @@
 import type { RequestMethod } from '@/entities/spec/requests'
 
+/**
+ * HTTP methods in a specific order
+ * Do not change the order
+ */
```
Comment (antlio) [inline packages/oas-utils/src/helpers/httpMethods.ts:6]: changing order in requests.ts won't impact the order to i changed it here + comment and kept previous code along @amritk

Comment (antlio) [reply packages/oas-utils/src/helpers/httpMethods.ts:6]: yes we kinda do as we add properties per method, but could be refactored

**Review by cameronrohani — APPROVED** [review]

**Review by amritk — APPROVED** [review]

**Pull request merged**